Bio/GenBank/Scanner.py

```diff
@@ -611,11 +615,11 @@ def parse_footer(self):
                 repr(self.line)
             # Remove tailing number now, remove spaces later
             linersplit = line.rsplit(None, 1)
-            if(len(linersplit) > 1):
+            if len(linersplit) > 1:
```

Comment:
Whomever wrote that must have done a lot of C code ;)

I wonder why we'd not spotted it during the PEP8 cleanup work?